controllers/provider-aws/pkg/controlplane/actuator.go

```diff
@@ -16,153 +16,37 @@ package controlplane
-					ClusterName:  clusterName,
-					APIServerURL: common.KubeAPIServerDeploymentName,
-				},
-			},
-			&secrets.ControlPlaneSecretConfig{
-				CertificateSecretConfig: &secrets.CertificateSecretConfig{
-					Name:       cloudControllerManagerServerName,
-					CommonName: cloudControllerManagerDeploymentName,
-					DNSNames:   controlplane.DNSNamesForService(cloudControllerManagerDeploymentName, clusterName),
-					CertType:   secrets.ServerCert,
-					SigningCA:  cas[gardencorev1alpha1.SecretNameCACluster],
-				},
-			},
-		}
-	},
-}
-
-var configChart = &chart.Chart{
-	Name: "cloud-provider-config",
-	Path: filepath.Join(aws.InternalChartsPath, "cloud-provider-config"),
-	Objects: []*chart.Object{
-		{
-			Type: &corev1.ConfigMap{},
-			Name: common.CloudProviderConfigName,
-		},
-	},
-}
-
-var ccmChart = &chart.Chart{
-	Name:   "cloud-controller-manager",
-	Path:   filepath.Join(aws.InternalChartsPath, "cloud-controller-manager"),
-	Images: []string{common.HyperkubeImageName},
-	ValuesFunc: func(clusterName string, shoot *gardenv1beta1.Shoot, checksums map[string]string) (map[string]interface{}, error) {
-		return map[string]interface{}{
-			"cloudProvider": "aws",
-			"clusterName":   clusterName,
-
-			"kubernetesVersion": shoot.Spec.Kubernetes.Version,
-			"podNetwork":        extensionscontroller.GetPodNetwork(shoot),
-			"replicas":          extensionscontroller.GetReplicas(shoot, 1),
-			"podAnnotations": map[string]interface{}{
-				"checksum/secret-cloud-controller-manager":        checksums[cloudControllerManagerDeploymentName],
-				"checksum/secret-cloud-controller-manager-server": checksums[cloudControllerManagerServerName],
-				"checksum/secret-cloudprovider":                   checksums[common.CloudProviderSecretName],
-				"checksum/configmap-cloud-provider-config":        checksums[common.CloudProviderConfigName],
-			},
-			"configureRoutes": false,
-			"environment": []map[string]interface{}{
-				{
-					"name": "AWS_ACCESS_KEY_ID",
-					"valueFrom": map[string]interface{}{
-						"secretKeyRef": map[string]interface{}{
-							"key":  aws.AccessKeyID,
-							"name": common.CloudProviderSecretName,
-						},
-					},
-				},
-				{
-					"name": "AWS_SECRET_ACCESS_KEY",
-					"valueFrom": map[string]interface{}{
-						"secretKeyRef": map[string]interface{}{
-							"key":  aws.SecretAccessKey,
-							"name": common.CloudProviderSecretName,
-						},
-					},
-				},
-			},
-			"resources": map[string]interface{}{
-				"limits": map[string]interface{}{
-					"cpu":    "500m",
-					"memory": "512Mi",
-				},
-			},
-		}, nil
-	},
-	Objects: []*chart.Object{
-		{Type: &corev1.Service{}, Name: "cloud-controller-manager"},
-		{Type: &appsv1.Deployment{}, Name: "cloud-controller-manager"},
-	},
-}
-
 // NewActuator creates a new Actuator that acts upon and updates the status of ControlPlane resources.
-func NewActuator() controlplane.Actuator {
+func NewActuator(helper *internal.Helper) controlplane.Actuator {
```

Comment:
Once splitting up the `Helper` is done, you don't need to embed this anymore.

Reply:
See my comment above.